### ISSUE TITLE
Mist ETH wallet network / deposit issue 

### ISSUE THREAD

**Issue by madhusudhan237** [ethereum/mist#3647]
HI there.
I installed latest version (as of 30 JAN 2018) MIST (Ethereum wallet) on MAC book. There seems a wallet could be created in 4 types of network ie;

1. Main network
2. Ropsten Network
3. Rikeby network
4. solo network

when i installed on my MAC, by default, the network chosen was "Rikeby network". Before even i realised this, i transferred 0.5 ETH from Kraken exchange to MIST wallet address (Rikeby network).
Now, the question is - Is it ok that i transfer ETH to wallet on Rikeby network (or) should ETH be sent only to other networks? If so, which network should i choose?
PS: I also realize that, my MIST blocks are lagging behind etherscan.io blocks (due to my slow Wifi).

I am relatively new for this Concepts. 

Thanks for your quick help.

OS & Version: Osx 
Number of blocks synchronized: approx 150000

**Comment by Taledeyo** [comment]
I am also new.

**Comment by WanderingBeing** [comment]
I'm not sure what Rikeby network is, but I know Ropsten is a test net and wallet addresses made there do not appear on the main net so I would say it is either in limbo on the net somewhere, or it bounced back to the exchange

**Comment by pathfinder2104** [comment]
I am also new and I am trying to learn to deploy ICO on ethereum... I faced this issue of no ether in the wallet, I tried several faucets but it didn't work. I was on Rikeby Test Net.  Then I switched to Ropsten Test Net and started mining. (Strange that with Rikeby I didn't get any option of start mining) within 50 or so odd minutes I got 375 ethers. I found this issue when I was searching for no ether in ethereum wallet and hence I am posting it here. It may help.
